java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -843,6 +843,49 @@ public void sismember(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void sunionstore(BaseClient client) {
+        String key1 = "{key}-1-" + UUID.randomUUID();
+        String key2 = "{key}-2-" + UUID.randomUUID();
+        String key3 = "{key}-3-" + UUID.randomUUID();
+        String key4 = "{key}-4-" + UUID.randomUUID();
+        String key5 = "{key}-5-" + UUID.randomUUID();
+
+        assertEquals(3, client.sadd(key1, new String[] {"a", "b", "c"}).get());
+        assertEquals(3, client.sadd(key2, new String[] {"c", "d", "e"}).get());
+        assertEquals(3, client.sadd(key4, new String[] {"e", "f", "g"}).get());
+        // create new
+        assertEquals(5, client.sunionstore(key3, new String[] {key1, key2}).get());
+        assertEquals(Set.of("a", "b", "c", "d", "e"), client.smembers(key3).get());
+        // overwrite existing set
+        assertEquals(5, client.sunionstore(key2, new String[] {key3, key2}).get());
+        assertEquals(Set.of("a", "b", "c", "d", "e"), client.smembers(key2).get());
+        // overwrite source
```

Comment:
Don't believe you need these test where you overwrite keys. It's not bad to have them I agree but doesn't seem like they had tests like these in Node and PY. They have more concise IT tests.

Reply:
@acarbonetto Opinion?